src/moj/components/cookie-banner/template.njk

```diff
@@ -0,0 +1,24 @@
+{%- from "govuk/components/button/macro.njk" import govukButton %}
+
+<div class="cookie-banner govuk-width-container cookie-banner--visible" role="region" aria-label="cookie banner">
+    <div class="govuk-grid-row">
+        <div class=" govuk-grid-column-two-thirds">
+            <div class="cookie-banner__message">
+                <h2 class="govuk-heading-m">{{ params.heading.title }}</h2>
+                <p class="govuk-body">{{ params.heading.html }}</p>
```

Comment:
Bit of an alignment with GOV.UK DS: allows `text` or `html` and allows `html` to have include HTML tags.
```suggestion
                <p class="govuk-body">{{ params.heading.html | safe if params.heading.html else params.heading.text }}</p>
```

Reply:
Added!